Resources/Audio/Misc/attributions.yml

```diff
@@ -47,3 +47,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from Citadel Station 13"
   source: "https://github.com/Skyrat-SS13/Skyrat13/commit/2d4f2d1b489590b559e4073f41b126cef56f4c50"
+
+- files: ["yellow.ogg"]
+  license: "Public Domain"
+  copyright: "Audio is posted by youtube user Making Contact, original audio from Emergency!"
+  source: "https://www.youtube.com/watch?v=T3uvP86W5bE"
```

Comment:
This needs a proper licence and it coming from a youtube video does not mean it is public domain. And even if the original series is public domain by now that does not mean that the sound effect they used for filming is so, too.